wiki/osu!wiki_Contribution_Guide/GitHub_Desktop/zh.md

```diff
@@ -1,105 +1,105 @@
+2\. 当安装结束后，GitHub Desktop 会提示你登陆，点击 `Sign into GitHub.com`。
 
-3\. Fill in the prompts as they appear.
+3\. 填写账号密码。
 
-4\. Continue to [Cloning](#cloning).
+4\. 下一步： [克隆分支](#cloning)。
 
-## Cloning
+## 克隆分支
 
-1\. Click on `Clone a repository`.
+1\. 点击 `Clone a repository`。
 
-2\. This will show a list of your repos; select the `osu-wiki` repo.
+2\. 这里会显示你的所有分支，选择 `osu-wiki` 仓库。
 
-3\. Select the clone location. It doesn't matter where you clone the repo to, just remember where you cloned it to.
+3\. 选择保存的位置。放在哪里都可以，请记住你保存的位置。
 
 ![](img/github-desktop-select-repo.jpg)
 
-4\. Click `Clone`. This may take a while depending on your network environment.
+4\. 点击 `Clone`。耐心等待一会儿，等待时间取决于你的网络状况。
 
-5\. Continue to [Branching](#branching).
+5\. 下一步：[分支](#branching)。
 
-## Branching
+## 修改分支名称
 
-*Note: This is technically optional, but is heavily recommended due to reasons that you will need to look up for yourself.*
+*提示：这一步是可选步骤，但是非常建议修改你的分支的名称，以便以后使用。*
 
-1\. Click on the `Current branch` dropdown. This is located at the top of the menu.
+1\. 在菜单的顶部，点击 `Current branch` 下拉菜单。
 
-2\. Enter a name for this branch. It is suggested to use the name of the article you are editing as the branch name followed by a number. For example, editing the `Forums` article may result in the branch being called `forums1`.
+2\. 为这个分支输入一个名字。建议使用你正在修改的文章的名字，后面接一个数字。比如，编辑 `Forums` 文章，可以使用分支名 `forums1`。
 
 ![](img/github-desktop-branch.jpg)
 
-3\. Continue to [Editing](#editing).
+3\. 下一步：[编辑](#editing)。
 
-## Editing
+## 编辑
 
-1\. In the folder you cloned the repo into, navigate to the file you want to edit. The articles are organized by the folder names being the article names and the `.md` files being the locales.
+1\. 在你已经克隆下来的仓库的文件夹中，找到你想要修改的文件。这些文章以文章标题作为文件夹名称，以 `.md` 为后缀的文件作为语言文件。
 
-2\. Once you find the correct file, open it using your favorite text editor.
+2\. 当你找到文件后，用你最喜欢的文本编辑器打开它。
 
-3\. You can now make your changes. When making changes, try to follow the [Article Styling Criteria](/wiki/ASC) as closely as you can.
+3\. 你现在可以修改该文件。当你编辑时，请尽可能遵循 [文章风格指南](/wiki/ASC)。
 
-4\. Continue to [Committing and pushing](#committing-and-pushing).
+4\. 下一步 [提交和推送](#committing-and-pushing)。
 
-### Moving existing files
+### 移动文件
 
-1\. Use Windows Explorer to move files around.
+1\. 使用 Windows 资源管理器移动文件。
 
-2\. Continue to [Committing and pushing](#committing-and-pushing).
+2\. 下一步 [提交和推送](#committing-and-pushing)。
 
-### Creating new files
+### 创建新文件
 
-*Caution: If you are going to create article files using Windows Explorer, make sure the `File name extensions` option is enabled.* See [How to show or hide file name extensions in Windows Explorer](https://support.microsoft.com/en-us/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer) for instructions.
+*注意：如果你在 Windows 资源管理器中创建新的文件，请确认 `文件扩展名` 选项已勾选。* 更多介绍：[如何显示或隐藏文件扩展名](https://support.microsoft.com/en-us/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer)
```

Comment:
https://support.microsoft.com/zh-cn/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer